server/src/db/migrations/20231106152320_createReviewRatings.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.table("reviews", (table) => {
+    table.integer("rating").notNullable().unsigned();
```

Comment:
`unsigned` is not necessary here, instead we can rely on model level validations